Octokit/Http/HttpClientAdapter.cs

```diff
@@ -81,6 +81,7 @@ protected virtual async Task<IResponse> BuildResponse(HttpResponseMessage respon
                 AcceptHeaders.RawContentMediaType,
                 "application/zip" ,
                 "application/x-gzip" ,
+                "zip" , // Not a standard MIME type but see issue #2898
```

Comment:
While I am not super excited about not using the proper representation (asper the mdm web docs and iana) - I get the situation here.  Thank you for making sure this is commented with context. ❤️